scripts/run-build.ps1

```diff
@@ -44,7 +46,7 @@ $env:CHANNEL=$env:RELEASE_SUFFIX
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows"
+    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows\$Architecture"
```

Comment:
I have done the update to `DotNetCli.cs` in this commit.

Right now we going to support x86 only for Windows. I do not want to add an `architecture` option to `run-build.sh` when it is not going to be used at all.